docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,692 @@
+---
+title: Internationalization (i18n) - Strapi Developer Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# 🌍  Internationalization (i18n)
+
+The Internationalization (i18n) plugin allows Strapi users to create, manage and distribute localized content in different languages, called "locales". For more information about the concept of internationalization, please refer to the
+[W3C definition](https://www.w3.org/International/questions/qa-i18n.en#i18n).
+
+The i18n plugin:
+
+- allows admin panel users to create several localized versions of their content (see [user guide](/user-docs/latest/content-manager/translating-content.md))
+- allows developers to build localized projects by fetching and consuming the right content depending on the country/language of the audience.
+
+::: tip NOTE
+The i18n plugin does not automatically translate the users' content nor adapt the admin interface to languages specificities (e.g. displaying the admin panel in Right To Left format).
+:::
+
+## Installation
+
+::: warning PREREQUISITES
+The Internationalization plugin is installed by default on all Strapi applications running on version 3.6.0 or higher. For lower versions, a migration is needed (see [Update Strapi version](/developer-docs/latest/update-migration-guides/update-version.md)), as well as a manual installation of the plugin.
+:::
+
+The plugin can be installed:
+
+- [from the Marketplace](/user-docs/latest/plugins/installing-plugins-via-marketplace.md),
+- or using the Terminal, by running one of the following commands:
+
+:::: tabs
+
+::: tab yarn
+
+``` bash
+yarn strapi install i18n
+```
+
+:::
+
+::: tab npm
+
+``` bash
+npm run strapi install i18n
+```
+
+:::
+
+::::
+
+## Usage with Strapi Content API
+
+The i18n plugin adds new features to the [Content API](/developer-docs/latest/developer-resources/content-api/content-api.md):
+
+- a new [`locale`](#getting-localized-entries-with-the-locale-parameter) parameter to fetch content only for a specified locale
+- the ability to create a localized entry, either [from scratch](#creating-a-new-localized-entry) or [for an existing localized entry](#creating-a-localization-for-an-existing-entry)
+
+### Getting localized entries with the `locale` parameter
+
+The `locale` [API parameter](/developer-docs/latest/developer-resources/content-api/content-api.md#api-parameters) can be used to fetch entries only for a specified locale. It takes a locale code as value (see [full list of available locales](https://github.com/strapi/strapi/blob/features/i18n/packages/strapi-plugin-i18n/constants/iso-locales.json)).
+
+:::tip TIP
+ To fetch content for a locale, make sure it has been already [added to Strapi in the admin panel](/user-docs/latest/settings/managing-global-settings.md#configuring-internationalization-locales).
+:::
+
+The format for a GET request is the following:
+
+```js
```

Comment:
I don't think js would be the right "language" but I'm also not familiar with any language tag in vuepress for these logical statements